### ISSUE TITLE
Investigate Alternative Top-Section Background

### ISSUE THREAD

**Issue by dzoladz** [alaoweb/2020.alaoweb.org#69]
The top-section abstract background image should be replaced with something more akin to the color palette of the ALAO website at https://www.alaoweb.org

**Comment by dwgreen1** [comment]
I like these two SVG backgrounds that I came across, customized using ALAO colors. Any thoughts on these?

**Option 1**
![option-1](https://user-images.githubusercontent.com/2590810/85563736-a81b4680-b5fb-11ea-9f8a-e8f87f318d65.png)

**Option 2**
![Screen Shot 2020-06-24 at 09 17 08](https://user-images.githubusercontent.com/2590810/85563765-ae112780-b5fb-11ea-8235-a56a2eb405b2.png)

**Comment by kenirwin** [comment]
Ooh! I like both of these. I might lean more towards the first one -- the tiling reminds me of a carpet-tile pattern and that makes it feel kind of "conference-y" to me. But I have no objection to the swoopy one either.

**Comment by dzoladz** [comment]
Yeah... both of these are great options. Option 1 does have more of a "conference-y" feel, but Option 2 has more of a calming affect. If pressed on the issue, I'd go with Option 2.

Others?

**Comment by dwgreen1** [comment]
I love them both, so we need a tie breaker.

**Comment by ssisler** [comment]
I think both work, but I'm leaning toward option 1.  The waves are nice and soothing, but they leave a bit of open space in the top right.

(Also, hi everyone!)

**Comment by hillmk1** [comment]
I agree that they're both good, but I prefer Option 2.

**Comment by dzoladz** [comment]
@Walton485 or @Nassaubound ?

**Comment by Nassaubound** [comment]
I could go either way.....but I think I like the 2nd 1 better.

**Comment by dwgreen1** [comment]
I think I am going to call it and choose option two.

**Comment by Walton485** [comment]
Option 2 for sure, in my case!
